fnv.go

```diff
@@ -1,8 +1,12 @@
 package bigcache
 
+import(
+	xxhash "github.com/cespare/xxhash"
```

Comment:
Do we need an alias for import?

Reply:
Hi, I'm not sure how to do this in github. You mind sharing what do you mean by alias for import? I mean, can you just mention to me what I need to do?